src/fsatrace.c

```diff
@@ -120,18 +132,24 @@ main(int argc, char *const argv[])
 		      "   m: dump file move operations\n"
 		      "   d: dump file delete operations\n"
 		      "   q: dump file stat operations\n"
+					"\n"
+					"  and buf_size must be a valid number of bytes\n"
 		      ,argv[0]);
 	out = argv[2];
-	if ((err = shmInit(&shm, out, LOGSZ, 1)))
+	if ((err = shmInit(&shm, out, buf_size, 1)))
 		fatal("allocating shared memory (%d)", err);
 	snprintf(envout, sizeof(envout), ENVOUT "=%s", out);
 	putenv(envout);
+	snprintf(envbufsize, sizeof(envbufsize), ENVBUFSIZE "=%ld", buf_size);
+	putenv(envbufsize);
 #ifdef _WIN32
 	// Workaround, bash distributed with ghc 8.6.5 seems to discard most
-	// environment variables, pass FSAT_OUT as the first PATH component.
-	snprintf(envout, sizeof(envout), "PATH=%s;%s", out, getenv("PATH"));
+	// environment variables, pass environment variables as the first few
+	// PATH components.
+	char		env    [65536];
```

Comment:
This is introducing a declaration in the middle, should probably be surrounded in `{}`.

Reply:
Done.